### PR TITLE
add awie open ticket and dsm

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Migrate awie dsm and openticket from old repo (release-22.10.0-next) to new repo

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by lgcosta — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

File: centreon-awie/.github/workflows/dependabot_jira.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: Create Dependabot Ticket on Jira
```
Comment (sc979) [inline centreon-awie/.github/workflows/dependabot_jira.yml:1]: Could you delete this file ? If not, we may delete it in another PR

File: centreon-dsm/.github/workflows/dependabot_jira.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+name: Create Dependabot Ticket on Jira
```
Comment (sc979) [inline centreon-dsm/.github/workflows/dependabot_jira.yml:1]: Could you delete this file ? If not, we may delete it in another PR

File: centreon-open-tickets/.github/workflows/dependabot_jira.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+name: Create Dependabot Ticket on Jira
```
Comment (sc979) [inline centreon-open-tickets/.github/workflows/dependabot_jira.yml:1]: Could you delete this file ? If not, we may delete it in another PR

File: centreon-awie/.github/workflows/dependabot_jira.yml (hunk 1 above)
Comment (tuntoja) [reply centreon-awie/.github/workflows/dependabot_jira.yml]: done

File: centreon-dsm/.github/workflows/dependabot_jira.yml (hunk 2 above)
Comment (tuntoja) [reply centreon-dsm/.github/workflows/dependabot_jira.yml]: done

File: centreon-open-tickets/.github/workflows/dependabot_jira.yml (hunk 3 above)
Comment (tuntoja) [reply centreon-open-tickets/.github/workflows/dependabot_jira.yml]: done

**Review by sc979 — APPROVED** [review]

**Pull request merged**